examples/context automation.ipynb

```diff
@@ -189,7 +189,7 @@
     "        entity_queries[entity][query] = entity_queries[entity][query].pipe(\n",
     "            add_threat_intel,\n",
     "            correlations=[get_ti_pubs],\n",
-    "            tenant_id=TENANT,\n",
+    "            tenant.id=TENANT,\n",
```

Comment:
I believe this will still be `tenant_id` as it is the function parameter and not a parsed column from a result set.

https://github.com/secureworks/taegis-magic/blob/main/taegis_magic/pandas/context.py#L492